test/ExecutionFramework.cpp

```diff
@@ -142,6 +142,11 @@ void ExecutionFramework::sendMessage(bytes const& _data, bool _isCreation, u256
 		entry.data = fromHex(log.data, WhenError::Throw);
 		m_logs.push_back(entry);
 	}
+
+	if (!receipt.status.empty())
```

Comment:
Note that this will silently produce wrong results for buggy versions of ``cpp-ethereum``.
Do you think it makes sense to assert that ``status`` is non-empty if the EVM version is >= byzantium? This would break all tests with buggy versions of ``cpp-ethereum``.